tests/snapshots/import_system__imports_alias_shadow.bend.snap

```diff
@@ -3,4 +3,5 @@ source: tests/golden_tests.rs
 input_file: tests/golden_tests/import_system/imports_alias_shadow.bend
 ---
 [4m[1m[31mErrors:[0m
+[1mDuring execution :[0m
 The import 'lib/nums/two' shadows the imported name 'lib/nums/one'
```

Comment:
Same thing for these other import errors.

Reply:
I think I'll change this so no message appears when there are only diagnostics with `source.file == None`, and when there are there are diagnostics with source files and ones without, the ones without will show in the end with the message `Other diagnostics:` or something similar. What do you think?